modules/installation-requirements-user-infra.adoc

```diff
@@ -151,7 +151,7 @@ ifndef::ibm-power[|4]
 ifndef::openshift-origin[]
 |Compute
 ifdef::ibm-z,ibm-power[|{op-system}]
-ifndef::ibm-z,ibm-power[|{op-system} or {op-system-base} 7.9 ^[2]^]
+ifndef::ibm-z,ibm-power[|{op-system}, {op-system-base} 7.9, or {op-system-base} 8.2 and later ^[2]^]
```

Comment:
Hi, may I know the reason we write `8.2 and later` here, since RHEL-8.4 was GA on 2021-05-18, so only RHEL-8.4 was scheduled in the RHEL-8 scale-up testing.
And also I noticed we emphasized RHEL-8.4 should be used in the rhel-images-aws chapter. 

```
When creating a {op-system-base} compute machine for AWS, ensure that the AMI is {op-system-base} 8.4 
```

Reply:
@gpei this is a really good question I've been trying to get an answer for. I was unsure what version of RHEL 8.y we were going to support. If QE is only testing RHEL 8.4, I will update this accordingly.

@mtnbikenc does that sound good from your perspective?

Reply:
Stating 8.4 is fine with me.  I've personally only done installs with 8.3 and 8.4.  Our CI testing for RHEL 8 workers currently uses 8.3 because of the [bug](https://bugzilla.redhat.com/show_bug.cgi?id=1977984) with 8.4 and NetworkManager-cloud-setup.